tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py

```diff
@@ -10,54 +12,128 @@ def __init__(self, *args, **kwargs):
+    def test001_create_cloudspace_with_wrong_params(self):
         """ ZRT-OVC-001
-        *Test case for ...*
+        *Test case for creating acloudspace with wrong parameters .*
 
         **Test Scenario:**
+        #. Create an cloudspace with providing non existing parameter, should fail.
+        #. Create an cloudspace with non-exist account , should fail.
+        #. Create an cloudspace with providing non existing parameter, should fail.
 
-        #. Create an account and 2 cloudspaces, should succeed.
-        #. check that the cloudspaces have been created.
         """
         self.log('%s STARTED' % self._testID)
+     
+        self.log('Create an cloudspace with providing non existing parameter, should fail')
+        self.cloudspaces[self.cs1] =  {'account': self.acc1, self.random_string(): self.random_string()}
+        res = self.create_cs(openvcloud=self.openvcloud, accounts=self.accounts, vdcusers=self.vdcusers,
+                             cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
+        self.assertEqual(res,'parameter provided is wrong' )
 
-        self.cs2 = self.random_string()
-        self.vdcuser = self.random_string()
-        self.vdcusers.extend([{self.vdcuser: {'openvcloud': self.openvcloud,
-                                              'provider': 'itsyouonline',
-                                              'email': 'abdelmab@greenitglobe.com'}}])
-        self.cloudspaces.extend([{self.cs2: {'account': self.acc1,
-                                             'users': OrderedDict([('name', self.vdcuser),
-                                                                   ('accesstype', 'CXDRAU')])}}])
-        self.temp_actions = {'account': {'actions': ['install']},
-                             'vdcuser': {'actions': ['install']}, 'vdc': ['install']}
+        self.log('Create an cloudspace with non-exist account , should fail')
+        self.cloudspaces[self.cs1] =  {'account': self.random_string}
+        res = self.create_cs(openvcloud=self.openvcloud, accounts=self.accounts, vdcusers=self.vdcusers,
+                             cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
+        self.assertEqual(res,"account doesn't exist" )
 
-        self.log('Create 1 account and 2 cloudspaces, should succeed')
-        self.create_cs(openvcloud=self.openvcloud, vdcusers=self.vdcusers, accounts=self.accounts,
+        self.log('Create an cloudspace without providing account parameter, should fail')
+        self.cloudspaces[self.cs1] = {}
+        res = self.create_cs(openvcloud=self.openvcloud, vdcusers=self.vdcusers, accounts=self.accounts,
                        cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
+        self.assertEqual(res, 'account is required')
+
+        self.log('%s ENDED' % self._testID)
 
-        # wait till blueprint is executed
-        self.wait_for_service_action_status(self.cs2)
+    @parameterized.expand("one", "two")   
```

Comment:
no need to make it for one Cloudspace .. it is the same logic used .. if it works for 2, it will work for one

Reply:
but it doesn't work for two , so we skip the creation of both

Reply:
u already check one cloudspace creation in anther test cases in the vms file .. so it is okay to make this testcase for 2 cloud spaces and skip it